interp/cfg.go

```diff
@@ -1510,24 +1510,29 @@ func (interp *Interpreter) cfg(root *node, pkgID string) ([]*node, error) {
 			for i := l - 1; i >= 0; i-- {
 				c := clauses[i]
 				c.gen = nop
-				body := c.lastChild()
-				if len(c.child) > 1 {
-					cond := c.child[0]
-					cond.tnext = body.start
-					if i == l-1 {
-						setFNext(cond, n)
+				if len(c.child) == 0 {
+					c.tnext = n
+					c.fnext = n
+				} else {
```

Comment:
this could be:
```
if len(c.child) == 0 {
    c.tnext = n
    c.fnext = n
    continue
}
````
and then you don't need the else case, and you can unindent again the whole block.